### PR TITLE
Update all slack invite links to redirect to the prefect.io slack link

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Replaces Slack invite links with a centralized link to prefect.io; removes redirects to heroku app.

### CODE REVIEW

File: src/components/Schematics/Schematic-Flow.vue [hunk 1]
```diff
@@ -1075,9 +1075,7 @@ export default {
               Thanks! We'll get our team on it immediately.
             </div>
             By the way, you can
-            <a
-              target="_blank"
-              href="https://join.slack.com/t/prefect-community/shared_invite/enQtODQ3MTA2MjI4OTgyLTliYjEyYzljNTc2OThlMDE4YmViYzk3NDU4Y2EzMWZiODM0NmU3NjM0NjIyNWY0MGIxOGQzODMxNDMxYWYyOTE"
+            <a target="_blank" href="https://prefect.io/slack"
```
Comment (whimsicallyson) [inline src/components/Schematics/Schematic-Flow.vue:1078]: any reason we don't want to use `ExternalLink` for this?

File: src/pages/Support.vue [hunk 2]
```diff
@@ -178,9 +178,7 @@ export default {
                       If you haven't already,
                     </span>
 
-                    <a
-                      target="_blank"
-                      href="https://join.slack.com/t/prefect-community/shared_invite/enQtODQ3MTA2MjI4OTgyLTliYjEyYzljNTc2OThlMDE4YmViYzk3NDU4Y2EzMWZiODM0NmU3NjM0NjIyNWY0MGIxOGQzODMxNDMxYWYyOTE"
+                    <a target="_blank" href="https://prefect.io/slack"
```
Comment (whimsicallyson) [inline src/pages/Support.vue:181]: same question re ExternalLink

**Review by whimsicallyson — APPROVED** [review]
this lgtm! a couple questions, but more for my curiosity than it not working as written.

File: src/components/Schematics/Schematic-Flow.vue (hunk 1 above)
Comment (znicholasbrown) [reply src/components/Schematics/Schematic-Flow.vue:1078]: Hm no reason, just that this predates the external link component and there'd be more styling to do here than I want atm

File: src/pages/Support.vue (hunk 2 above)
Comment (znicholasbrown) [reply src/pages/Support.vue:181]: Same as above

**Pull request merged**